### PR TITLE
chore: substitute Travis with Github Actions

### PR DESCRIPTION
**Pull request by CodeWithSouma**
@mariobehling  @CloudyPadmal  
## Substitute Travis with Github Actions, #2148 
* remove travis.yml
* add android.yml

### CODE REVIEW

**Comment by CodeWithSouma** [comment]
@CloudyPadmal  sir can you help regarding this problem #2149. how to fix this ...

**Comment by CloudyPadmal** [comment]
Hello @CodeWithSouma , Travis build is not currently working for the repos.. That's why the build is failing. So just ignore that for now. Let me check the updates you made.

**Comment by anurag0singh** [comment]
Why was this rejected?

**Comment by anurag0singh** [comment]
@CloudyPadmal Sir why was the Github CI is rejected in this particular merge request?

**Review by CloudyPadmal — APPROVED** [review]
Looks good to me. Let's go ahead with this and see..

**Comment by CloudyPadmal** [comment]
@anurag0singh Travis CI has no more free credits to use by the repos. So we are moving to GitHub actions

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Hi @CodeWithSouma ,

In Travis CI build, when a PR is merged into development branch, an APK file is auto generated. Same happens when the development branch is merged to main branch. Shall we add a function like that to GitHub actions?

**Comment by CodeWithSouma** [comment]
> Hi @CodeWithSouma ,
> 
> In Travis CI build, when a PR is merged into development branch, an APK file is auto generated. Same happens when the development branch is merged to main branch. Shall we add a function like that to GitHub actions?

I am trying to implement that thing

**Comment by CloudyPadmal** [comment]
@CodeWithSouma Perfect! Thank you..

**Comment by CodeWithSouma** [comment]
@CloudyPadmal sir, I add the jobs in GitHub actions, an APK file that is auto-generated.

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build debug APK
+        run: bash ./gradlew assembleDebug --stacktrace
+      - name: Upload APK
+        uses: actions/upload-artifact@v1
+        with:
+          name: app
+          path: app/build/outputs/apk/debug/app-debug.apk
```
Comment (CloudyPadmal) [inline .github/workflows/android.yml:39]: It's not simple as that. We use two build flavours and once built, a script is used to upload the files to apk branch. https://github.com/fossasia/pslab-android/blob/development/.travis.yml#L37

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by CloudyPadmal** [comment]
Also, can you test the action setup in a demo branch? Then we can see if it's building without any error.

**Comment by kartikeysaran** [comment]
@CodeWithSouma are you still working on this ?

**Comment by CodeWithSouma** [comment]
> @CodeWithSouma are you still working on this ?

yes

**Comment by kartikeysaran** [comment]
> > @CodeWithSouma are you still working on this ?
> 
> yes

Okay ! 🍻

**Review by CloudyPadmal — APPROVED** [review]
I'll merge this and let's move forward with improvements.

**Pull request merged**